### ISSUE TITLE
SIRvb's Image Adjustments uses alt text to apply css classes, reducing accessibilty

### ISSUE THREAD

**Issue by MasssiveJuice08** [morrowind-modding/morrowind-modding.github.io#36]
The [syntax of SIRvb's Image Adjustments CSS Snippet ](https://publish.obsidian.md/slrvb-docs/ITS+Theme/Image+Adjustments#Syntax) uses Markdown embedded image alt text to apply CSS classes to images. The basic principle is the same as [Obsidian-flavored-Markdown's syntax for resizing images](https://help.obsidian.md/Linking+notes+and+files/Embed+files#Embed+an+image+in+a+note).

By occupying the alt text field, it prevents us from applying actual alt text to images. Alt text has many benefits:
- alt text is useful for displaying an image description if the image cannot load for any reason
- it enables sight-impaired persons to read the image description using a screenreader (don't tell me no one is going to make Morrowind accessible to sight-impaired persons. I'd bet money someone in the future makes this possible in OpenMW)
- it improves website SEO by telling sitecrawlers what an image is.

This could possibly be resolved by making a provision for two alt text fields for images; one that gets read as CSS classes, and another as alt text.

**Comment by MasssiveJuice08** [comment]
Possible alternative is to wrap images in HTML `<div>` tags - https://quartz.eilleeenz.com/Quartz-Snippets#applying-custom-styling-through-writing-markdown

This would have the added benefit of also solving Obsidian and Quartz's parsing of alt arrtibute keywords: If adding an image that has 'banner' or 'profile' in the file name, the CSS currently picks this up as an alt text key word and styles it accordingly, even though the keywords aren't actually present in the alt text, they're only in the file name. This will trip up users who don't understand why their image is forced to display like a banner or a profile picture.

- [ ] Check whether standard Markdown image formatting will work inside the `<div>` - alt text, embed etc.
